src/tests/test_solve_postfix_equation.py

```diff
@@ -78,7 +87,9 @@
     ("0 _ 1 + 0 cos +", 2),
     ("984526.123654 _ 100 _ + 1 sin - 0 cos *", 1001.3914269),
     ("987654322 2 / 827161 - 7000000 +", 500000000),
-    ("-987654322 2 / 827161 - 7000000 -", -500000000)
+    ("-987654322 2 / 827161 - 7000000 -", -500000000),
+    ("987654322 2 / 827161 - 7000000 + 0 ^ 1 +", 500000000),
+    ("-987654322 2 / 827161 - 7000000 - 43 ^ 2 -", -500001849)
```

Comment:
mocnina by měla být ve tvaru v postfixu "x(základ) y(exponent) ^", ty to máš infixově "x(základ) ^ y(exponent)"